examples/get_authcode.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import platform
+import sys
+from bleak import BleakClient, BleakScanner
+from bleak.exc import BleakError
+from re import S
+
+from pyfreshintellivent import FreshIntelliVent
```

Comment:
Some linting issues. Remove unused imports + remove empty line.
```suggestion
import asyncio
import sys
from bleak import BleakClient, BleakScanner
from pyfreshintellivent import FreshIntelliVent
```